src/utils/index.test.js

```diff
@@ -1076,3 +1084,37 @@ describe('getGrowingPhase', () => {
     expect(getGrowingPhase(crop)).toEqual(phase)
   })
 })
+
+describe('unlockTool', () => {
```

Comment:
Great idea to add tests for this!

Reply:
i was embarrassed to find that they were missing :speak_no_evil: